docs/version-2-upgrade.md

```diff
@@ -0,0 +1,385 @@
+---
+subcategory: ""
+page_title: "Runtime Connector Module Version 2 Upgrade Guide"
+description: |-
+  Runtime Connector Module Version 2 Upgrade Guide
+---
+
+# Runtime Connector Module Version 2 Upgrade Guide
+
+Version 2.0.0 of the Runtime Connector Module a major release and includes some changes that you will need to consider when upgrading. This guide is intended to help with that process.
+
+Upgrade topics:
+
+- [Runtime Connector Module Version 2 Upgrade Guide](#runtime-connector-module-version-2-upgrade-guide)
+  - [Module Version Configuration](#module-version-configuration)
+  - [Removed Inputs: `addons` and `addon_params`](#removed-inputs-addons-and-addonparams)
+    - [Refactoring the `aws/secretsmgr` Addon](#refactoring-the-awssecretsmgr-addon) 
+    - [Refactoring the `aws/kinesis-firehose` Addon](#refactoring-the-awskinesis-firehose-addon) 
+    - [Refactoring the `aws/kinesis-data-stream` Addon](#refactoring-the-awskinesis-data-stream-addon) 
+  - [Removed Input: `custom_external_id`](#removed-input-customexternalid)
+  - [Removed Output: `account_id`](#removed-output-accountid)
+  - [Removed Output: `settings`](#removed-output-settings)
+  - [New output: `sym_integration`](#new-output-symintegration)
+  - [New output: `sym_runtime`](#new-output-symruntime)
+
+## Module Version Configuration
+Before upgrading to version 2.0.0 or later, it is recommended to upgrade to the most recent 1.X version of the module (version 1.0.6) 
+and ensure that your environment successfully runs `terraform plan` without unexpected changes.
+
+It is recommended to use [version constraints when configuring Terraform providers](https://www.terraform.io/docs/configuration/providers.html#provider-versions). 
+If you are following that recommendation, update the version constraints in your Terraform configuration and run [`terraform init -upgrade`](https://www.terraform.io/docs/commands/init.html) to download the new version.
+
+For example, given this previous configuration:
+
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 1.0"
+
+  environment = var.environment
+}
+```
+
+An updated configuration would be:
+
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 2.0"
+
+  environment = var.environment
+}
+```
+
+Optionally, you may choose to use `moved` configuration blocks to migrate your Terraform state, instead of replacing the existing AWS IAM Role and Policy with a new one:
```

Comment:
```suggestion
Optionally, you may also use `moved` configuration blocks to migrate your Terraform state, instead of replacing the existing AWS IAM Role and Policy with a new one:
```
Nit: Clarify that this is something you do in addition to the version change, not instead of the version change. Might also be good to link to the Terraform docs that explain how `moved` blocks work?